doc/source/cluster/aws-tips.rst

```diff
@@ -73,10 +95,199 @@ You should see something like
 Please refer to `this discussion <https://github.com/ray-project/ray/issues/9327>`__ for more details.
 
 
+.. _aws-cluster-efs:
+
+Amazon EFS
+~~~~~~~~~~
+
+To use Amazon EFS, install some utilities and mount the EFS in ``setup_commands``. Note that these instructions only work if you are using the AWS Autoscaler.
+
+.. note::
+
+  You need to replace the ``{{FileSystemId}}`` to your own EFS ID before using the config. You may also need to set correct ``SecurityGroupIds`` for the instances in the config file.
+
+.. code-block:: yaml
+
+    setup_commands:
+        - sudo kill -9 `sudo lsof /var/lib/dpkg/lock-frontend | awk '{print $2}' | tail -n 1`;
+            sudo pkill -9 apt-get;
+            sudo pkill -9 dpkg;
+            sudo dpkg --configure -a;
+            sudo apt-get -y install binutils;
+            cd $HOME;
+            git clone https://github.com/aws/efs-utils;
+            cd $HOME/efs-utils;
+            ./build-deb.sh;
+            sudo apt-get -y install ./build/amazon-efs-utils*deb;
+            cd $HOME;
+            mkdir efs;
+            sudo mount -t efs {{FileSystemId}}:/ efs;
+            sudo chmod 777 efs;
+
+
+.. _aws-cluster-emr:
+
+Amazon EMR
```

Comment:
Can you remove this and the following sagemaker section? These are different deployment models. 

It's not enough for these examples to just work at a given point in time. Adding them to the docs creates an implication of supporting those deployment models, which is a rather large decision.

AWS can provide its own support for running Ray on EMR, but Ray's would only be providing first-class support via the manual cluster setup steps (e.g. the public `ray start`/`ray stop` apis). 

If you feel strongly about trying to get first class support for EMR, I think the right next steps would be to open up a [REP (ray enhancement proposal) ](https://github.com/ray-project/enhancements)